librz/core/rop.c

```diff
@@ -396,6 +401,29 @@ RZ_IPI RzRopRegInfo *rz_core_rop_reg_info_dup(RzRopRegInfo *src) {
 	return dup;
 }
 
+/**
+ * \brief Find the RzRopRegInfo in the dependencies
+ * \param gadget_info Pointer to the RzRopGadgetInfo
+ * \param name Register name
+ *
+ * \return RzList* *<RzRopRegInfo *>
```

Comment:
The `\return` should describe what the content of `RzRopRegInfo` actually means (e.g.: "the found RzRopRegInfo structs which for \p name which are dependencies" or something like this.). The returned type can be easily seen from the declaration.
Also for the parameters. What are the parameters actually used for?

Reply:
Agreed :+1: , edited